WcaOnRails/app/models/competition.rb

```diff
@@ -920,6 +920,25 @@ def set_wcif_events!(wcif_events, current_user)
     reload
   end
 
+  # Takes an array of partial Person WCIF and updates the fields that are not immutable.
+  def update_persons_wcif!(wcif_persons, current_user)
+    persons_schema = { "type" => "array", "items" => User.wcif_json_schema }
+    puts persons_schema.inspect
+    JSON::Validator.validate!(persons_schema, wcif_persons)
+
+    ActiveRecord::Base.transaction do
+      wcif_persons.each do |wcif_person|
+        # registration = registrations.joins(:user).where("users.id = ?", wcif_person["wcaUserId"]).first
```

Comment:
Vestigial?

Reply:
Yeah, I did even figure out a better query than both this and the one below.